elasticsearch/elasticsearch_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/venicegeo/pz-gocommon/gocommon"
 )
 
-const MOCKING = true
+const MOCKING = false
```

Comment:
You need to switch this back before committing (otherwise Jenkins will freak out)